### DISCUSSION TITLE
Unity documentation and examples are lacking and maybe postpone after June 28

### DISCUSSION THREAD

**Discussion by iulianbroasca** [WalletConnect/walletconnect-monorepo#2625] [Developer Support]
Hey, 

Given that the documentation and examples are lacking and servers will be down 2 weeks from now, is it a chance for postponing shutting down the server after June 28 because most of the people have encountered some problems during migration from v1 to v2? 

Without good documentation and examples the migration/integration of V2 is quite complicated for the Unity community. 

In my case, I managed to test it on Unity Editor and it worked fine but on mobile everything crashed, and right now I'm stuck here and I believe that what is going on is out of our control now.

Thank you,
Iulian

**Comment by finessevanes** [comment]
@iulianbroasca our unity team has provided this README for support: https://github.com/WalletConnect/WalletConnectUnity/blob/2.0/README.md

**Comment by mustafadervisglu** [comment]
@finessevanes You have prepared an amateurish SDK and honestly, it's a disappointment for me. How could you degrade such a good project like this?

**Comment by yozdev** [comment]
Hello @finessevanes

It seems v2 update was not properly tested on Unity before v1 shutdown. It is affecting most of us who were using v1.

Our game is already live and now we do not have any answers for our player community when they ask us about the WalletConnect error fixes.

Request a swift resolution.
